exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -115,6 +118,7 @@ public UserSession build() {
 
   private UserSession() {
     queryCount = new AtomicInteger(0);
+    sessionId = nextSessionId.getAndIncrement();
   }
```

Comment:
Thanks Arina. Just realized you are the author of #666 :) Could you please tell me how I can merge your changes into my branch. Am new to this process. Thanks.